tools/Nethermind.SendBlobs/Program.cs

```diff
@@ -184,10 +202,40 @@
         string txRlp = Hex.ToHexString(txDecoder
             .Encode(tx, RlpBehaviors.InMempoolForm | RlpBehaviors.SkipTypedWrapping).Bytes);
 
+        BlockModel<Keccak>? blockResult = null;
+        if (waitForBlobInclusion)
+            blockResult = await nodeManager.Post<BlockModel<Keccak>>("eth_getBlockByNumber", "latest", false);
+       
         string? result = await nodeManager.Post<string>("eth_sendRawTransaction", "0x" + txRlp);
 
         Console.WriteLine("Result:" + result);
-
         nonce++;
+
+        if (txCount > 0 && blockResult != null && waitForBlobInclusion) 
+            await WaitForBlobInclusion(nodeManager, tx.CalculateHash(), blockResult.Number);    
+    }
+}
+
+async Task WaitForBlobInclusion(INodeManager nodeManager, Keccak txHash, UInt256 lastBlockNumber)
+{
+    Console.WriteLine("Waiting for blob transaction to be included in a block");
+
+    while (true)
+    {
+        var blockResult = await nodeManager.Post<BlockModel<Keccak>>("eth_getBlockByNumber", lastBlockNumber, false);
+        if (blockResult != null)
+        {
+            lastBlockNumber++;
+            
+            if (blockResult.Transactions.Contains(txHash))
+            {
+                Console.WriteLine($"Found blob transaction in block {blockResult.Number}");
+                return;
+            }
+        }
+        else
+        {
+            await Task.Delay(2000);
+        }
     }
 }
```

Comment:
Could also run `dotnet format whitespace . --folder` in the project folder and change `var` to types?

Reply:
Changed as suggested